quic-client/src/quic_client.rs

```diff
@@ -180,3 +180,9 @@ impl ClientConnection for QuicClientConnection {
         Ok(())
     }
 }
+
+pub(crate) fn close_quic_connection(connection: Arc<QuicClient>) {
```

Comment:
Could this be abusable if my server never confirms connection close and blocks runtime indefinitely here?

Reply:
No. the closed.await will return LocallyClosed immediately.  To be sure, I added a unit test test_connection_close which close after the server is shutdown. No delay observed.